install/configuring_inventory_file.adoc

```diff
@@ -1002,18 +1002,21 @@ can be deployed using the installer.
 
 If no registry storage options are used, the default OpenShift Container
 Registry is ephemeral and all data will be lost when the pod no longer exists.
+
+[IMPORTANT]
+====
+Testing has shown issues with using the NFS server on RHEL as storage backend for the container registry. This includes the embedded registry and Quay, Cassandra for metrics storage, and ElasticSearch for logging storage. Therefore, using NFS on RHEL as storage for these framework services is not recommended.
```

Comment:
We can remove the references to Cassandra, and Elastic Search here.

The section this is in, implies we will not be discussing those components.

Reply:
I removed these in the new PR.